<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StructuredGridRegionSelectWidget</class>
 <widget class="QWidget" name="StructuredGridRegionSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>132</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>195</width>
    <height>132</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="iMinLabel">
     <property name="text">
      <string>I Min:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="SliderWithValue" name="iMinSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="iMaxLabel">
     <property name="text">
      <string>I Max:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="SliderWithValue" name="iMaxSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="jMinLabel">
     <property name="text">
      <string>J Min:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="SliderWithValue" name="jMinSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="jMaxLabel">
     <property name="text">
      <string>J Max:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="SliderWithValue" name="jMaxSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderWithValue</class>
   <extends>QSlider</extends>
   <header>guibase/widget/sliderwithvalue.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
